freqtrade/rpc/rpc.py

```diff
@@ -281,10 +281,11 @@ def _rpc_balance(self, fiat_display_currency: str) -> Dict:
                 rate = 1.0
             else:
                 try:
-                    if coin in('USDT', 'USD', 'EUR'):
-                        rate = 1.0 / self._freqtrade.get_sell_rate('BTC/' + coin, False)
+                    pair = self._freqtrade.exchange.get_valid_pair_combination(coin, "BTC")
+                    if pair.startswith("BTC"):
+                        rate = 1.0 / self._freqtrade.get_sell_rate(pair, False)
                     else:
```

Comment:
Need to think a bit about logic imlemented here...

Reply:
maybe to explain: 
we want the currency in BTC so we can filter (I think it should be configurable, but let's leave that for another day)
so we either multiply by the rate (`rate * price` - or divide (`1/rate * price` or `price / rate`).

This logic is not new - just the filter when to use what method is...